### ISSUE TITLE
Rename theta and phi to azimuth and incident/takeoff/emergence

### ISSUE THREAD

**Issue by miili** [pyrocko/kite#97]
Naming the LOS angles theta and phi was a really awful idea.

We should use clear naming for the angles: azimuth and emergence

**Comment by hvasbath** [comment]
I agree, however, I would rather opt for heading instead of azimuth, emergence would be ok for me.

**Comment by miili** [comment]
Heading can be a little misleading since we are looking at the reference system (point of observation) of the satellite. Azimuth on the other hand implies a spherical coordinate system. Maybe @hsudha has an opinion on this?
